test/IDE/complete_override.swift

```diff
@@ -115,9 +115,6 @@ protocol ProtocolA {
   func protoAFunc()
   @objc optional func protoAFuncOptional()
 
-  @noreturn
-  func protoAFuncWithAttr()
-
```

Comment:
This was testing how functions with attributes show up in code completion; we need a replacement testcase.